src/frontend/apps/desk/src/features/members/components/MemberGrid.tsx

```diff
@@ -129,6 +131,7 @@ export const MemberGrid = ({ team, currentRole }: MemberGridProps) => {
       </Card>
       {isModalMemberOpen && (
         <ModalAddMembers
+          currentRole={currentRole}
```

Comment:
I feel the same applies for the `currentRole`. The `currentRole` is computed based on team's data in `[id].tsx` and passed down to the last element. Two questions, should we use the `team.abilities` directly to avoid wrapper? Should we compute the current role with a function wherever it's needed, so avoid passing it through components?

Reply:
Yes this one could be refacto to be put in a function. We have to modify lot of tests though, let's do it in another PR.